scripts/generate.sh

```diff
@@ -10,14 +12,14 @@ then
 
     echo "Setting up swc_ecma_ast $SWC_AST_VERSION... provide --quick to just code generate"
     rm -rf swc_ecma_ast
-    cargo clone --vers $SWC_AST_VERSION swc_ecma_ast
-    (cd swc_ecma_ast && cargo +nightly rustdoc -- --output-format json)
+    cargo clone --verbose --vers $SWC_AST_VERSION swc_ecma_ast
+    (cd swc_ecma_ast && cargo +$NIGHTLY_VERSION rustdoc --verbose -- --output-format json)
     cp swc_ecma_ast/target/doc/swc_ecma_ast.json swc_ecma_ast.json
 
     echo "Setting up swc_ecma_parser $SWC_PARSER_VERSION..."
     rm -rf swc_ecma_parser
-    cargo clone --vers $SWC_PARSER_VERSION swc_ecma_parser
-    (cd swc_ecma_parser && cargo +nightly rustdoc -- --output-format json)
+    cargo clone --verbose --vers $SWC_PARSER_VERSION swc_ecma_parser
+    (cd swc_ecma_parser && cargo +$NIGHTLY_VERSION rustdoc --verbose -- --output-format json)
```

Comment:
@magurotuna I think you just need to change this to:

```
(cd swc_ecma_parser && cargo +nightly rustdoc -- --output-format json -Z unstable-options)
```

Then you can remove all the nightly_version stuff. I'd rather this always be using the latest nightly and we can keep updating progressively.

Reply:
That fixes the issue! Thanks😃